pkg/legacy/keps/proposals.go

```diff
@@ -70,12 +69,15 @@ func (p *Parser) Parse(in io.Reader) *api.Proposal {
 		proposal.Error = errors.Wrap(err, "error unmarshaling YAML")
 		return proposal
 	}
-	if err := validations.ValidateStructure(test); err != nil {
-		proposal.Error = errors.Wrap(err, "error validating KEP metadata")
-		return proposal
-	}
 
-	proposal.Error = yaml.UnmarshalStrict(metadata, proposal)
+	/*
+		if err := validations.ValidateStructure(test); err != nil {
+			proposal.Error = errors.Wrap(err, "error validating KEP metadata")
+			return proposal
+		}
+	*/
```

Comment:
@justaugustus - why this was commented out? We no longer validate e.g. if PRR approvers are from PRR approvers, etc....

@jeremyrickard 

@johnbelamaric @deads2k - FYI

Reply:
@wojtek-t -- Discussing on [#enhancements](https://kubernetes.slack.com/archives/C1L57L91V/p1612288049074600) and will send an updated PR later.

Thanks for flagging this!